client/me/help/main.jsx

```diff
@@ -224,54 +219,6 @@ class Help extends PureComponent {
 		);
 	};
 
-	supportSessionCard = () => {
-		const { translate, hasAppointment, scheduleId, isUserBlocked } = this.props;
-
-		//If we already have an appointment or the scheduleId has not been loaded, bail
-		if ( hasAppointment || null === scheduleId || isUserBlocked ) {
-			return;
-		}
-
-		return (
-			<Card className="help__support-session-card">
```

Comment:
We seem to be leaving some redundant styles in the linked stylesheet. Let's remove the unused `help__support*` set of styles ...

Reply:
Thanks for catching this!